param_templates/input_nml.yaml

```diff
@@ -47,6 +47,10 @@ diag_manager_nml:
             else: 30
     max_axes:
         values: 90
+    auto_merge_nc:
+        values:
+            $OCN_GRID in ["tx2_3v2", "tx0.25v1"]: .true.
+            else: .false.
```

Comment:
Can we use the `description` field in this file? Or just add comments? I'm assuming you only want to set `auto_merge_nc = .true.` for grids with a reasonable default for  `TARGET_IO_PES`, but it would be nice to have a comment either confirming that or explaining why these are the only two grids that use `auto_merge_nc` out of the box